starboard/android/shared/android_main.cc

```diff
@@ -310,6 +311,7 @@ Java_dev_cobalt_coat_StarboardBridge_startNativeStarboard(JniEnvExt* env) {
 #if SB_IS(EVERGREEN_COMPATIBLE)
   StarboardThreadLaunch();
 #else
+  starboard::shared::starboard::GetLoggingMutex();
```

Comment:
Was this getting used before being initialized? I can't see where that would happen